### ISSUE TITLE
WebUI not connecting after install

### ISSUE THREAD

**Issue by Brouwertje** [rogerfar/rdt-client#611]
**What version are you using?**
Tteck script
**Wat OS are you running?**
Proxmox
**Are you using Docker or as a service?**
No
**Which debrid provider are you using?**
Real Debrid
**Which downloader are you using?**
n/a
**Please attach a log file here with the log setting set to debug**

Whatever I do, IP:6500 is not working. I'm using the script. Tried on other Proxmox device, same issue. 
Never had any problems with other LXC containers. RDTclient is the only one that does not work.

**Comment by DevMan01** [comment]
I noticed something similar, here are my logs... but I'm on Ubuntu:

> Nov 19 00:42:07 ubuntu-server-01 RdtClient[952]: To install missing framework, download:
Nov 19 00:42:07 ubuntu-server-01 RdtClient[952]: https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=9.0.0&ar
Nov 19 00:42:07 ubuntu-server-01 systemd[1]: rdtc.service: Main process exited, code=exited, status=150/n/a
Nov 19 00:42:07 ubuntu-server-01 systemd[1]: rdtc.service: Failed with result 'exit-code'.

I'm going to tinker around and see if updating my .net version does the trick. I noticed in the latest release that the framework update was in there.

**Comment by Brouwertje** [comment]
Ah yes

i see:

root@rdtclient:~# journalctl -xe
Nov 19 09:29:28 rdtclient RdtClient[6688]: To install missing framework, download:
Nov 19 09:29:28 rdtclient RdtClient[6688]: https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=9.0.0&arch=x64&rid=linux-x64&os=debian.12
Nov 19 09:29:28 rdtclient systemd[1]: rdtc.service: Main process exited, code=exited, status=150/n/a
░░ Subject: Unit process exited
░░ Defined-By: systemd
░░ Support: https://www.debian.org/support
░░ 
░░ An ExecStart= process belonging to unit rdtc.service has exited.
░░ 
░░ The process' exit code is 'exited' and its exit status is 150.
Nov 19 09:29:28 rdtclient systemd[1]: rdtc.service: Failed with result 'exit-code'.
░░ Subject: Unit failed

**Comment by DevMan01** [comment]
Yep, this was my problem too. 

Except I wasn't running as a container, I was hard installed on Ubuntu 22. I had to upgrade to 24, then install .net 9 - but... I had my .net 8 installed via snap. So, I had to uninstall snap .net 8, then install .9 over that. Then I had to update my rdtc service in /etc/systemd/system/ so it wasn't calling the snap `dotnet` executable, but the one from /usr/local/bin instead.

Hope this helps anyone who ran into the same problem.
